en/operations/admin-procedures.html

```diff
@@ -121,6 +121,217 @@ <h2 id="status-pages">Status pages</h2>
 
 
 
+<h2 id="schema-changes">Schema changes</h2>
+<p>
+  Refer to <a href="../reference/schema-reference.html#modifying-schemas">modifying schemas</a>
+  for reference documentation for making changes to a schema, and actions required.
+  Find procedures below for a subset of such changes.
+</p>
+
+
+<h3 id="attribute-to-index">Attribute to index</h3>
+<p>
+  Changing between <code>index</code> and <code>attribute</code> is a common field change operation
+  to optimize performance.
+  Use the <a href="../reindexing.html">reindexing</a> feature to safely migrate data to/from index structures.
+</p>
+<p>
+  When the attribute aspect of a field is removed, the attribute data is dropped on next process restart.
+  Hence, doing a direct change from attribute to index is risky, as a node stop can incur field data loss.
```

Comment:
😱

Reply:
Not really data loss though, but a gap in availability?

Reply:
will test again to qualify this, could be I did this wrong

Reply:
you are right, @bratseth  - the data survived a proton restart even though not yet reindexed from attribute into index. after subsequent deployment, which triggered the actual reindex, the index was populated. Will rewrite to availability not loss in next PR. Thanks!